sn_protocol/src/storage/scratchpad.rs

```diff
@@ -35,10 +37,11 @@ pub struct Scratchpad {
 
 impl Scratchpad {
     /// Creates a new instance of `Scratchpad`.
-    pub fn new(owner: PublicKey) -> Self {
+    pub fn new(owner: PublicKey, version: u64) -> Self {
         Self {
             address: ScratchpadAddress::new(owner),
             encrypted_data: Bytes::new(),
+            version,
```

Comment:
I wonder if this `version` is confusing w/ the counter there. 

Probably one name should change. 

maybe this is `data_encoding` or something? 🤔

Reply:
Make sense, data_encoding is excellent